astropy/io/fits/src/compressionmodule.c

```diff
@@ -979,27 +1133,28 @@ PyObject* compression_compress_hdu(PyObject* self, PyObject* args)
 
 PyObject* compression_decompress_hdu(PyObject* self, PyObject* args)
 {
+
     PyObject* hdu;
     tcolumn* columns = NULL;
 
     void* inbuf;
     size_t inbufsize;
 
-    PyArrayObject* outdata;
+    PyArrayObject* outdata = NULL;
     int datatype;
     int npdatatype;
     npy_intp zndim;
-    npy_intp* znaxis;
+    npy_intp* znaxis = NULL;
     long arrsize;
-    unsigned int idx;
+    int idx;
 
     fitsfile* fileptr = NULL;
     int anynul = 0;
     int status = 0;
 
-    if (!PyArg_ParseTuple(args, "O:compression.decompress_hdu", &hdu))
-    {
-        PyErr_SetString(PyExc_TypeError, "Couldn't parse arguments");
+    int free_columns_manually = 1;
```

Comment:
Not a big deal, but `<stdbool.h>` provides `bool`, `true`, and `false`.

Reply:
I had some problems with them in C extensions, some Python versions (at least on windows) are bound to C90.

Reply:
😞